ansible_base/activitystream/serializers.py

```diff
@@ -28,11 +29,11 @@ class Meta:
     related_content_type_model = serializers.SerializerMethodField()
     changes = serializers.SerializerMethodField()
 
-    def get_content_type_model(self, obj):
+    def get_content_type_model(self, obj) -> str:
```

Comment:
I disagree with this type-hint. What if `not obj.content_type`?

Reply:
also, wait, `str`? That can't be right

Reply:
You have 1 test that returned `None` by artificially creating an content type. That's fine to test but that's not realistic.

Reply:
What if the model class goes away entirely in a future migration? That's what the test is really testing, iirc. And that seems like a realistic situation, even if rare.

Reply:
ContentTypes are not cleaned up when removing a model, intended by design.

These type hints produce documentation for _API clients_. If a model class goes away, and the app removes its content type, yet for some reason doesn't remove the activity stream entries, then that's a bug of the app developer.

I believe it's a bad decision, but I will make this change.

Reply:
I mean I'm also fine with just removing the conditional if we're _that_ confident. But I think the type signature should match the code (and really that should be enforced by the type checker, but alas)